images/nodesvc-base/Dockerfile

```diff
@@ -1,20 +1,43 @@
-FROM ghcr.io/scality/federation/nodesvc-base:7.10.5.0
-
-ENV UTAPI_CONFIG_FILE=${CONF_DIR}/config.json
+ARG DEBIAN_FRONTEND=noninteractive
+ARG UTAPI_VERSION=latest
+FROM ghcr.io/scality/utapi:${UTAPI_VERSION} AS base
+
+FROM node:22.11.0-bookworm-slim
```

Comment:
We should be able to use `node:22.14` in this repo as well, even for artesca and the CI, I don't see any big changes in the changelog

Reply:
using `node:22.14.0-bookworm-slim` now, on both images.